### PR TITLE
Don't show error if no intro.md was found

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Intro page

### Description
This PR changes the behaviour of the intro page. If no intro could be fetched then the output should be blank.

### Steps

- [x] Added implementation
- [x] Added / updated tests
- [x] I read the [contribution documentation](https://github.com/hedgedoc/react-client/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

**Review by DerMolly — APPROVED** [review]

File: public/mock-backend/public/screenshot.png.license [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
```
Comment (DerMolly) [inline public/mock-backend/public/screenshot.png.license:1]: Didn't we determine that this is unnecessary?

Comment (mrdrogdrog) [reply public/mock-backend/public/screenshot.png.license]: ARGHH! rebase issues everywhere -_-

Comment (mrdrogdrog) [reply public/mock-backend/public/screenshot.png.license]: fixed

**Comment by DerMolly** [comment]
You also seem to got an empyt `public/mock-backend/public/img/demo.png.license` in this PR.

**Comment by mrdrogdrog** [comment]
...

you know what? I'll fix that later when the base branch is merged.

**Review by DerMolly — APPROVED** [review]

File: src/components/intro-page/intro-page.tsx [hunk 2]
```diff
@@ -22,33 +22,34 @@ import { WaitSpinner } from '../common/wait-spinner/wait-spinner'
...
+      <p className="lead">
+        <Trans i18nKey="app.slogan"/>
+      </p>
+      <div className={ 'mb-5' }>
+        <Branding delimiter={ false }/>
+      </div>
+      <CoverButtons/>
+      <ShowIf condition={ !rendererReady && introPageContent !== undefined }>
+        <WaitSpinner/>
+      </ShowIf>
```
Comment (ErikMichelson) [inline src/components/intro-page/intro-page.tsx:41]: Why did you remove the brackets around the returned content?

**Review by ErikMichelson — COMMENTED** [review]
Just noting:
The real backend should return 404 is no intro.md was found. Otherwise things like this start to happen:

image

File: src/components/intro-page/intro-page.tsx (hunk 2 above)
Comment (mrdrogdrog) [reply src/components/intro-page/intro-page.tsx]: Dunno. I reverted it.

**Review by ErikMichelson — APPROVED** [review]
